src/components/ux-select/ux-select.hbs

```diff
@@ -0,0 +1,8 @@
+<label>Select Box
+    <select>
+        <option value="husker">Husker</option>
```

Comment:
The selection values are hard coded in? These should come from the datamodel for this component!